src/simulation/dynamics/spinningBodies/spinningBodiesOneDOF/_UnitTest/test_spinningBodyOneDOFStateEffector.py

```diff
@@ -310,8 +310,8 @@ def test_spinning_body_enforces_limits(max_lim, min_lim):
     spinningBody.theta_max = np.deg2rad(max_lim)
     spinningBody.theta_min = np.deg2rad(min_lim)
 
-    with pytest.raises(ValueError):
-        spinningBody.Reset(0)
+    # with pytest.raises(ValueError) as exc:
```

Comment:
please split and squash these commits to the final version to clean up the commit history.